cmd/queue/main.go

```diff
@@ -201,8 +205,55 @@ func createAdminHandlers() *http.ServeMux {
 	return mux
 }
 
+func probeQueueHealthPath(port int) error {
+	url := fmt.Sprintf("http://127.0.0.1:%d%s", port, queue.RequestQueueHealthPath)
+
+	httpClient := &http.Client{
+		Transport: &http.Transport{
+			// Do not use the cached connection
+			DisableKeepAlives: true,
+		},
+		Timeout: 500 * time.Millisecond,
```

Comment:
Is this an end-to-end request timeout? If so, I think this is too short and doesn't make use of the queue-proxies mechanism to keep the connection on /health open for 10 seconds currently.

It feels like we might want these retries in only one place. Maybe we should strip them out of the prober itself and only have this health binary retry tightly?

Reply:
Put this up to 10 seconds. Makes sense to use the full 10 seconds on /health.

I don't understand all of the moving parts, but it seems like we may want to keep the retry logic in the prober. The function with the  prober (`probeUserContainer`) is also used by a handler on the `queueServingPort` when a request has a `K-Network-Probe` header. Do we care about retries there?